Sources/Momento/messages/responses/ResponseBase.swift

```diff
@@ -1,4 +1,4 @@
-public class ErrorResponseBase: ErrorResponseBaseProtocol, CustomStringConvertible {
+public class ErrorResponseBase: ErrorResponseBaseProtocol, Error, CustomStringConvertible {
```

Comment:
this change was to allow Momento errors to be rethrown as described [here](https://github.com/momentohq/dev-eco-issue-tracker/issues/604)